### PR TITLE
Fixing Sync API docs

### PR DESCRIPTION
**Pull request by rahuln3223**

### CODE REVIEW

File: api-reference/beta/resources/synchronization-stringkeylongvaluepair.md [hunk 1]
```diff
@@ -18,8 +18,8 @@ Represents a key-value pair where the key is a string and the value is an Int64.
 ## Properties
 | Property	   | Type	|Description|
 |:---------------|:--------|:----------|
-|key|String|Key.|
-|value|Int64|Value.|
+|key|String| The mapping of the user type from source to target.|
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/synchronization-stringkeylongvaluepair.md:21]: I notice samples in the tracker. Should we document them?
```suggestion
|key|String| The mapping of the user type from the source system to the target system. For example:<br/><li>`User to User` - For Azure AD to Azure AD synchronization <br/><li>`worker to user` - For Workday to Azure AD synchronization. <br/> |
```

File: api-reference/beta/resources/synchronization-synchronizationsecretkeystringvaluepair.md [hunk 2]
```diff
@@ -21,6 +21,15 @@ Represents a single secret value.
...
 |value|String|The value of the secret.|
 
+### Supported Key Value Pairs
+| Key	   |Value|
+|:---------------|:----------|
+|Base Address| The tenant URL / SCIM end point that you are trying to provision.|
+|SecretToken	| The token to authorize access to the application. This token is provided by the application developer. |
+|SyncNotificationSettings	|This property is used to determine if quarantine emails are sent.|
+|SyncAll	|Determines scope of sync between all users and groups or only ones that are assigned. When provisioning users into applications, the value is generally “True”.|
+|SkipOutOfScopeDeletions |Determines if accounts that go out of scope will/won’t be disabled in the target.|
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/synchronization-synchronizationsecretkeystringvaluepair.md:31]: This value isn't listed in the table above but it's valid. See my comment above.

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Hi @rahuln3223 - Thanks for handling these changes. I've left suggestions for fixes prior to approving the PR.

File: api-reference/beta/api/synchronization-synchronizationjob-validatecredentials.md [hunk 3]
```diff
@@ -44,6 +44,7 @@ In the request body, provide a JSON object with the following parameters.
 |:---------------|:--------|:----------|
 |useSavedCredentials|Boolean|When `true`, the `credentials` parameter will be ignored and the previously saved credentials (if any) will be validated instead. |
 |credentials|[synchronizationSecretKeyStringValuePair](../resources/synchronization-synchronizationsecretkeystringvaluepair.md) collection|Credentials to validate. Ignored when the `useSavedCredentials` parameter is `true`.|
+|templateId|String| Defines default settings for the provisioning configuration.|
```
[the remark this answers is not in the document]
Comment (rahuln3223) [reply api-reference/beta/api/synchronization-synchronizationjob-validatecredentials.md:47]: If not implemented, take it out of beta schema as well.
Or document it, and say not implemented.

**Review by FaithOmbongi — APPROVED** [review]
Thank you @rahuln3223. Approved!

**Pull request merged**